sdk/easm/defender-easm-rest/package.json

```
@@ -0,0 +1,131 @@
+{
+  "name": "@azure-rest/defender-easm",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "Defender EASM Client",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/defender-easm.d.ts",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/defender-easm.d.ts",
+    "README.md",
+    "LICENSE",
+    "review/*"
+  ],
+  "engines": {
+    "node": ">=14.0.0"
```

We just upgraded our min supported version to v16:
```suggestion
    "node": ">=16.0.0"
```